docs/src/appendix/cast/script/init.md

```diff
@@ -0,0 +1,17 @@
+# `init`
+Create script template and directories structure.
+
+The command creates the following structure inside `scripts` directory:
```

Comment:
It also creates the scripts directory itself, so perhaps we should reprase this a bit?